ignite/cmd/plugin.go

```diff
@@ -360,6 +364,130 @@ func NewPluginUpdate() *cobra.Command {
 	}
 }
 
+func NewPluginAdd() *cobra.Command {
+	cmdPluginAdd := &cobra.Command{
+		Use:  "add [path]",
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return fmt.Errorf("must provide a plugin path")
+			}
+			chain, err := newChainWithHomeFlags(cmd)
+			s := cliui.New(cliui.WithStdout(os.Stdout))
+
+			if err != nil {
+				return err
+			}
```

Comment:
```suggestion
			chain, err := newChainWithHomeFlags(cmd)
			if err != nil {
				return err
			}
			s := cliui.New(cliui.WithStdout(os.Stdout))			
```

I prefer to have the error check right after.